src/management/HooksManager.js

```diff
@@ -0,0 +1,368 @@
+var ArgumentError = require('rest-facade').ArgumentError;
+var utils = require('../utils');
+var Auth0RestClient = require('../Auth0RestClient');
+var RetryRestClient = require('../RetryRestClient');
+
+/**
+ * Simple facade for consuming a REST API endpoint.
+ * @external RestClient
+ * @see https://github.com/ngonzalvez/rest-facade
+ */
+
+/**
+ * @class HooksManager
+ * The hook class provides a simple abstraction for performing CRUD operations
```

Comment:
```suggestion
* The hooks manager class provides a simple abstraction for performing CRUD operations
```

Reply:
I would have auto-made this change but your PR is not set to allow changes back to your fork.